scout/models/case/case_loading_models.py

```diff
@@ -466,18 +466,19 @@ def set_synopsis(cls, synopsis: Optional[Union[str, List]]) -> Optional[str]:
             synopsis = ". ".join(synopsis)
         return synopsis
 
-    @field_validator("genome_build", mode="before")
+    @model_validator(mode="before")
     @classmethod
-    def format_build(cls, genome_build: Union[str, int]) -> str:
+    def format_build(cls, values) -> "CaseLoader":
```

Comment:
Ah, ok, so this runs first, and coerces ints to str?! It takes a little longer to read fully, but it makes it very clear at a glance which type is preferred while coping with other values more silently. That is good Pydantic to me.

Reply:
Yes, it fixes it before the validation happens